web/js/projection/ui.js

```diff
@@ -1,4 +1,5 @@
 import $ from 'jquery';
+import 'jquery-ui/effect-slide';
```

Comment:
Good catch! I missed that when I `import`ed jquery UI.

Reply:
It was a tough one to find tbh.

Reply:
Yeah, I went through the entire jquery UI API docs and searched for each function and `import`ed the ones I found we were using, but I guess I missed one. 😬 Thanks for catching it!